### PR TITLE
Private Websocket Position V5

### PR DESCRIPTION
**Pull request by 0cv**
Implement Private V5 for Position.

It's inspired from the private V1 spot for outbound account info, but there are some some tweaks to get a better error handling when the connection is dead, in order to allow a reconnection (see README updates).

### CODE REVIEW

File: v5_client_web_socket_service.go [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+import "github.com/gorilla/websocket"
+
 // V5WebsocketServiceI :
 type V5WebsocketServiceI interface {
 	Spot() V5WebsocketSpotServiceI
 	Linear() V5WebsocketLinearServiceI
 	Inverse() V5WebsocketInverseServiceI
 	Option() V5WebsocketOptionServiceI
-	Private() V5WebsocketPrivateServiceI
+	Private() (V5WebsocketPrivateService, error)
```
Comment (hirokisan) [inline v5_client_web_socket_service.go:11]: 👍

File: v5_ws_private.go [hunk 2]
```diff
@@ -1,10 +1,256 @@
...
+	if err := s.connection.WriteMessage(websocket.TextMessage, buf); err != nil {
+		return err
+	}
+	return nil
+}
+
+// Start :
+type ErrHandler func(isWebsocketClosed bool, err error)
+
+func (s *V5WebsocketPrivateService) Start(errHandler ErrHandler, ctx context.Context) error {
```
Comment (hirokisan) [inline v5_ws_private.go:171]: nits:

context.Context should be the first parameter of a function

**Review by hirokisan — APPROVED** [review]
Thank you for taking the time to commit!

I have not thought deeply about whether the error returned by Start should handle the connection or errHandler, but it seems to be a good idea at this point.

File: v5_ws_private.go (hunk 2 above)
Comment (0cv) [reply v5_ws_private.go:171]: Right, it's been merged now, but I'll implement the Order websocket soon, so will take care of this in that upcoming PR

Comment (0cv) [reply v5_ws_private.go:171]: nevermind, I see you added a commit

**Comment by 0cv** [comment]
I don't have a strong opinion on this one. But a few thoughts:
- I've never been able to get an error returned by `Start`
- If an error were to happen in `Start`, and shall be handled, I'd think that it would be the `Ping()` which could fail. But I've never seen the `Ping()` failing, even with no connection (not sure why). Regardless of that, I'd imagine the errHandler to always be notified with `s.connection.ReadMessage()` eventually failing. That leads to my point where the errHandler could be called multiple times, which I don't think would be desirable.

Please let me know what you think

**Pull request merged**